elyra/pipeline/component_parser_kfp.py

```diff
@@ -72,7 +72,7 @@ def _parse_properties(self, component_yaml: Dict[str, Any]) -> List[ComponentPar
             type = param.get('type', 'string')
 
             # Set description and include parsed type information
-            description = self._get_description_with_type_hint(param.get('description'), type)
+            description = self._get_description_with_type_hint(type, param.get('description'))
```

Comment:
Per the previous comment about renaming the method and reordering parameters, it might be worthwhile adding a default value to the getter here.